src/Features/WetnessEffects.cpp

```diff
@@ -37,10 +50,122 @@ void WetnessEffects::DrawSettings()
 		ImGui::SliderFloat("Puddle Max Angle", &settings.PuddleMaxAngle, 0.0f, 1.0f);
 		ImGui::SliderFloat("Puddle Min Wetness", &settings.PuddleMinWetness, 0.0f, 1.0f);
 
+		ImGui::SliderFloat("Max Fog Power", &settings.FogPowerThreshold, 0.0f, 1.0f);
+		if (ImGui::IsItemHovered()) {
+			ImGui::BeginTooltip();
+			ImGui::PushTextWrapPos(ImGui::GetFontSize() * 35.0f);
+			ImGui::Text("The maxiumum Fog Power considered foggy weather. (Try raising this if foggy weather isn't causing wetness)");
```

Comment:
Looking at GitHub only so don't know if this wraps too much. If it does, break the string into multiple parts (e.g., "part1 " "part2 ", etc. Notice the space before the end quote. This should result in linting formating it properly and also displaying properly using native imgui wrapping.